src/mono/mono/mini/mini-ops.h

```diff
@@ -1757,17 +1757,17 @@ MINI_OP3(OP_ARM64_SQRDMLSH_SCALAR, "arm64_sqrdmlsh_scalar", XREG, XREG, XREG, XR
 
 #endif // TARGET_ARM64
 
-MINI_OP(OP_FCVTL, "convert_to_higher_precision", XREG, XREG, NONE)
-MINI_OP(OP_FCVTL2, "convert_to_higher_precision_2", XREG, XREG, NONE)
-MINI_OP(OP_USHLL, "unsigned_shift_left_long", XREG, XREG, IREG)
-MINI_OP(OP_USHLL2, "unsigned_shift_left_long_2", XREG, XREG, IREG)
-MINI_OP(OP_SSHLL, "signed_shift_left_long", XREG, XREG, IREG)
-MINI_OP(OP_SSHLL2, "signed_shift_left_long_2", XREG, XREG, IREG)
-MINI_OP(OP_SHL, "shl", XREG, XREG, IREG)
-MINI_OP(OP_SSHR, "sshr", XREG, XREG, IREG)
-MINI_OP(OP_USHR, "ushr", XREG, XREG, IREG)
-MINI_OP3(OP_USRA, "usra", XREG, XREG, XREG, IREG)
-MINI_OP3(OP_SSRA, "ssra", XREG, XREG, XREG, IREG)
+MINI_OP(OP_SIMD_FCVTL, "convert_to_higher_precision", XREG, XREG, NONE)
+MINI_OP(OP_SIMD_FCVTL2, "convert_to_higher_precision_2", XREG, XREG, NONE)
+MINI_OP(OP_SIMD_USHLL, "unsigned_shift_left_long", XREG, XREG, IREG)
+MINI_OP(OP_SIMD_USHLL2, "unsigned_shift_left_long_2", XREG, XREG, IREG)
+MINI_OP(OP_SIMD_SSHLL, "signed_shift_left_long", XREG, XREG, IREG)
+MINI_OP(OP_SIMD_SSHLL2, "signed_shift_left_long_2", XREG, XREG, IREG)
+MINI_OP(OP_SIMD_SHL, "shl", XREG, XREG, IREG)
+MINI_OP(OP_SIMD_SSHR, "sshr", XREG, XREG, IREG)
+MINI_OP(OP_SIMD_USHR, "ushr", XREG, XREG, IREG)
+MINI_OP3(OP_SIMD_USRA, "usra", XREG, XREG, XREG, IREG)
+MINI_OP3(OP_SIMD_SSRA, "ssra", XREG, XREG, XREG, IREG)
```

Comment:
shouldn't the string versions also start with simd_? like for the wasm ones below